taipy/rest/config/rest_config.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2021-2025 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+from typing import Dict, Optional, Tuple
+
+from taipy.common.config import Section, _inject_section
+
+
+class RestConfig(Section):
+    name: str = "REST"
+
+    def __init__(self):
+        super().__init__("rest")
+        self._port: int = 5000
+        self._host: str = "127.0.0.1"
+        self._use_https: bool = False
+        self._ssl_cert: Optional[str] = None
+        self._ssl_key: Optional[str] = None
+
+    def __copy__(self):
+        new_instance = RestConfig()
+        new_instance._port = self._port
+        new_instance._host = self._host
+        new_instance._use_https = self._use_https
+        new_instance._ssl_cert = self._ssl_cert
+        new_instance._ssl_key = self._ssl_key
+        return new_instance
+
+    @classmethod
+    def _clean(cls):
+        return RestConfig()
```

Comment:
The clean `method` is made to reset the current in-memory instance so the config does not hold any copy with obsolete attribute values. It should set all the attributes to their default values.

```python
    def _clean(self):
        self._port = self._DEFAULT_PORT
        self._port = self._DEFAULT_HOST
        ...
```